src/Assets/BenefitCategoryLists/cashAssistanceBenefits.tsx

```diff
@@ -25,6 +25,18 @@ const cashAssistanceBenefits = {
       defaultMessage="State cash assistance for individuals 60 years of age or older (Old Age Pension/OAP)"
     />
   ),
+  ssdi: (
+    <FormattedMessage
+      id="cashAssistanceBenefits.ssdi"
+      defaultMessage="Social security benefit for people with disabilities (Social Security Disability Insurance/SSDI)"
```

Comment:
We will need to add this new FM `id` and `defaultMessage` to the translations UI via api/translations/admin.